tensorflow/lite/micro/kernels/transpose_conv.cc

```diff
@@ -28,9 +28,11 @@ limitations under the License.
 namespace tflite {
 namespace {
 
-constexpr int kInputTensor = 0;
+// OutputShapeTensor is tensor 0, but it is unused in TFLM because we do not
+// support dynamic tensor resizing.
```

Comment:
move this comment next to `kOutputTensor=0`?

Reply:
Technically OutputShapeTensor is input tensor 0 so I wanted to put this comment where it would otherwise have been declared.

Reply:
Let's add some more context to the comment then. I confused it with kOutputTensor.

Something like:
```cc

// For the TfLite transpose_conv implementation, input tensor 0 corresponds to the
// OutputShapeTensor. However, since TFLM does not support dynamic tensors, the TFLM 
// implementation ignores input tensor 0 and the only inputs we care about are kFilterTensor, 
// kInputTensor and kBiasTensor.
```